subsys/logging/Kconfig.backends

```diff
@@ -298,10 +298,12 @@ config LOG_BACKEND_ADSP
 	  family of audio processors
 
 config LOG_BACKEND_FS
-	bool "Enable LittleFS backend"
+	bool "Enable file system backend"
 	depends on FILE_SYSTEM
 	help
-	  When enabled backend is using LittleFS to output logs.
+	  When enabled, backend is using the configured file system to output logs.
```

Comment:
I think that information should be added that logs gathered prior to mounting file system will not be saved.

Reply:
I added an additional sentence about that. Thanks.